src/app/components/dashboard-apps-dock/dashboard-apps-dock.coffee

```diff
@@ -0,0 +1,157 @@
+
+DashboardAppsDockCtrl = ($scope, $cookies, $uibModal, $window, MnoeOrganizations, MnoeAppInstances) ->
+  'ngInject'
+
+  $scope.appDock = {}
+  $scope.appDock.isMinimized = true
+  $scope.activeApp = null
+  $scope.launchApp = {isClosed: true}
+
+  # 'Lock' the dock when a menu is expanded.
+  # Ie: we disable all effects and animation
+  $scope.appDock.isAnimated = () ->
+    $scope.activeApp == null
+
+  # Scope initialization
+  $scope.modal ||= {}
+
+  # Mobile dock: max-height need to be an absolute value for the scrolling to work
+  pageHeight = jQuery(window).height()
+  navHeight = pageHeight * 0.7
+  jQuery('#mobile-dock').css({ maxHeight: navHeight + 'px' })
```

Comment:
In angular, DOM manipulation should be done in the link function: https://github.com/maestrano/mno-enterprise-angular/pull/38/files#diff-c12e2485499409ac1a745786e41fa594R102